src/art/FabianCristancho/script.js

```diff
@@ -0,0 +1,101 @@
+var canvas = document.getElementById('jack');
+var context = canvas.getContext('2d');
+// context.fillStyle = '#6666ff';
```

Comment:
Please remove outcommented code.

Reply:
thanks for the clarification. I just deleted the commented lines, so please check it :D